app/views/emails/EmailApiSubscriptionsView.scala.html

```diff
@@ -67,7 +67,7 @@
         </form>
          @if(apiVersionFilter.isDefined && !apiVersionFilter.get.isEmpty ) {
             <div><hr/></div>
-            @userTable(users, composeEmailRecipients, emails, applicationConfig, Map("API subscription" -> apiVersionFilter.get))
+            @userTable(users, composeEmailRecipients, emails, applicationConfig, Map("API subscription" -> apiVersions.filter(v => v.value == apiVersionFilter.get).head.description.trim))
```

Comment:
Do we know for sure that the description field will always be present?

Reply:
Yes, the description is what we see in the dropdown of API selection